### ISSUE TITLE
after/before?

### ISSUE THREAD

**Issue by kssfilo** [rborn/nativescript-i18n#17]
i18n doesn't work on following sequence.
1.tns prepare ios
2.open platforms/ios/*.xcodeproj by xcode
3.build by xcode
->L() doesn't work

4.tns prepare ios again
5.build by xcode
->L() works

Because Hook script creates lproj into app folder after
copying app folder to platforms folder by tns prepare

This means there are no lproj files on plaform folder at 1st
time.

lproj files will be copied from app folder to platform folder at 2nd "tns prepare".
so L() will work at 2nd build.

I tried to make hook script to before(refer my fork). and it seems works fine. 

I am not sure it makes other impact.and I haven't tested on Android.

**Comment by rborn** [comment]
Would you care to test on android too and make a PR 😺 ? I saw you also have some changes in the way the bundle is detected.

Thank you.

**Comment by kssfilo** [comment]
OK. I checked on android by following sequence. 
1.tns prepare android
2.build apk by gradlew manually and install it via adb
->L() doesn't work ,There are no value-ja in platforms folder (I tried Japanese)

3.tns prepare android again
4.build apk by gradlew
->L() works, (value-ja folder appears in platforms folder)

This looks same as ios. 

"before-prepare" version seems works fine both ios/android.

I will test few day and send PR if it is ok. Thanks.
